int32 fiducial_id
---
# No response fields required
